examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/RelaxActivity.java

```diff
@@ -0,0 +1,27 @@
+package gov.nasa.jpl.aerie.banananation.activities;
+
+import gov.nasa.jpl.aerie.banananation.Mission;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType.EffectModel;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.Export.Parameter;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.Export.Validation;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+
+import java.util.List;
+
+/**
+ * Relax? I needed something with no parameters this was the best one I could come up with
+ *
+ * @subsystem fruit
+ * @contact Jane Doe
+ */
+@ActivityType("Relax")
+public final class RelaxActivity {
+  private static final double RELAX_DURATION = 1.0;
+
+  @EffectModel
+  public void run(final Mission mission) {
+    //delay(Duration.of(RELAX_DURATION, Duration.HOURS));
```

Comment:
I think we can delete this line, as well as the `RELAX_DURATION` field.

Reply:
Sounds good. I wasn't sure about how to use `delay()` for this model so I included it as a placeholder but I can get rid of it. It doesn't seem like any of the activities in BananaNation have a duration, which I thought might be useful to test but I'm unsure how to do that.

Reply:
Removed!